docker/files/webapp.conf

```diff
@@ -0,0 +1,8 @@
+server {
+  listen 80;
+  server_name wagn;
```

Comment:
We'll need to use an environment variable, or something like that so we can make this universal.